tests/ert_tests/shared/test_main_entry.py

```diff
@@ -33,3 +33,47 @@ def test_main_logging_argparse(monkeypatch, caplog):
     with caplog.at_level(logging.INFO):
         main.main()
     assert "mode='test_run'" in caplog.text
+
+
+def test_api_database_url_forwarded(monkeypatch):
```

Comment:
While it is good to have tests. Just wondering if it's necessary to test that the parameters are passed correctly :shrug: Just asking :)

Reply:
I know, I just felt uneasy simply modifying without any sort of tests.. Don't like the fact either than after committing only the changes, another one (e.g. myself in some x number of days...) can come and remove those changes without any tests failing..

I would think a more unified testing of the command line parsing would be more appropriate though, couldn't really find anywhere else it was tested? (except through integration tests).

I would be more than happy to modify the tests to make them more valuable though!

Reply:
I've thought about it a bit, and I don't think it is excessive.. In fact, I would like to add tests to extensively verify that the parser is set up correctly, and that the right path is run given input arguments.